feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/grid/TimetableGrid.kt

```diff
@@ -79,6 +86,8 @@ fun TimetableGrid(
     timetable: Timetable,
     timeLine: TimeLine?,
     selectedDay: DroidKaigi2025Day,
+    isBookmarked: (TimetableItemId) -> Boolean,
+    onBookmarkClick: (TimetableItemId) -> Unit,
```

Comment:
The text displayed in the context menu must change depending on whether the item is bookmarked or not.
Therefore, it is impossible to implement this feature without modifying this part.
If the bookmark toggle operation cannot also be performed from the context menu, there is no point in submitting this PR, so removing `onBookmarkClick` is also impossible.